src/test/java/net/masterthought/cucumber/generators/AbstractPageTest.java

```diff
@@ -63,12 +64,15 @@ public void buildGeneralParametersAddsCommonProperties() {
 
         // then
         VelocityContext context = Deencapsulation.getField(page, "context");
-        assertThat(context.getKeys()).hasSize(5);
+        assertThat(context.getKeys()).hasSize(6);
         assertThat(context.get("jenkins_source")).isEqualTo(configuration.isRunWithJenkins());
         assertThat(context.get("jenkins_base")).isEqualTo(configuration.getJenkinsBasePath());
         assertThat(context.get("build_project_name")).isEqualTo(configuration.getProjectName());
         assertThat(context.get("build_number")).isEqualTo(configuration.getBuildNumber());
         assertThat(context.get("jenkins_source")).isNotNull();
+        assertThat(context.get("counter"))
+            .isNotNull()
+            .isInstanceOf(Counter.class);
```

Comment:
if you validate isIntanceOf you probably don't have to use IsNotNull at the same time

also validate value, not only the type, if deterministic